src/tr1d1um/getSync.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+type GetSync map[string][]string
+
+func (gs *GetSync) GetList(syncName string) (syncList []string, exists bool) {
+	if gs == nil || syncName == "" {
+		return nil, false
+	}
+	syncList, exists = (*gs)[syncName]
+	return
+}
```

Comment:
The sync list functionality didn't gain acceptance & can/should be removed.

Reply:
Sounds good.